LICENSE

```diff
@@ -0,0 +1,264 @@
+The Qemu TZ ARM bios contains work under the following licenses (these
+are extracted from the included code).
+
+A majority of the files are covered under a 2-point BSD-style license,
+with various copyright holders.
+
+Some files are covered under a 3-point BSD license.
+
+libfdt is under a dual GPL/2-point BSD-style license.
+
+Some files are released under public domain
+
+For definitive licensing declarations, please see the individual
+source files.
+
+----------------------------------------------------------------------
+
+bios/arm32.h:
```

Comment:
Should we detail here each file and each owner? This will become difficult to support and keep up-to -date. I would prefer to indicate there is GPLv2 / BSD2 / BSD3 / NEWLIB / dlmalloc public domain /  Smartall public domain license terms.

@jbech-linaro what do you think?